code/object/object.cpp

```diff
@@ -1300,24 +1374,28 @@ void obj_move_all_post(object *objp, float frametime)
 						color c;
 						weapon_get_laser_color(&c, objp);
 						light_color.set_rgb(&c);
+						light_color.set_rgb(fl2i(i2fl(light_color.r()) * r_mult), fl2i(i2fl(light_color.g()) * g_mult), fl2i(i2fl(light_color.b()) * b_mult));
```

Comment:
`set_rgb` twice? Should there be a condition here picking between the `weapon_get_laser_color` and the curve?